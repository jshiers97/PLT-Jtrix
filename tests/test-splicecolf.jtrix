void main() {
	Matrix<float> a;
	Matrix<float> b;
	float[] dim;
	int i;
	int j;

	a = [[1.2,2.3];[1.5,2.6]];
	b = a.spliceColumn_f(0);

	dim = b.dim_f();

	for(i = 0; i < f_to_int(dim[0]); i = i++) {
		for(j = 0; j < f_to_int(dim[1]); j = j++) {
			printf(b[i][j]);
		}
	}

	free(dim);
	free(b);
	free(a);

}
