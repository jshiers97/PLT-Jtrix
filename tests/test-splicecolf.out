2.300
2.600
